fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -44,6 +44,7 @@ public class IcebergTable extends Table {
 
     private static final String ICEBERG_CATALOG = "starrocks.catalog-type";
     private static final String ICEBERG_METASTORE_URIS = "iceberg.catalog.hive.metastore.uris";
+    private static final String ICEBERG_IMPL = "iceberg.catalog-impl";
```

Comment:
will this compatible with old version？

Reply:
There is no influence on other kinds of catalogs.